examples/README.md

```diff
@@ -56,20 +56,31 @@ To run this program, call it with a CPACS file, e.g.
 
     c-demo simpletest.cpacs.xml
 
-
-
-Python-Demo
------------
-
-The Python demo is similar to the C demo. In order to make it work, one needs to add 
-
-    TIGL_INSTALL_DIR/share/tigl3/python
-    TIXI_INSTALL_DIR/share/tixi3/python
-
-to the PYTHONPATH environment variable. Afterwards, test it with
-
-    python python_demo.py simpletest.cpacs.xml
-
+    
+Python-Demo {#lib_intrernal_python_demo}
+---------------------
+To run the python demo, you need to have the internal python API installed.
+The installation for Python API differ from the installation for the other languages.
+The easiest way to install TiGL 3 with the internal python api support is to use the
+amazing [conda](http://conda.pydata.org/miniconda.html) package manager. To install, type
+
+     conda create -n cpacscreator_env python=3.6 cpacscreator -c dlr-sc
+     activate cpacscreator_env
+    
+This installs TiGL and all its dependencies. 
+
+Then, you can run the python examples as usual. For example type:
+
+     python create_space_ship.py
+
+Make sure the script create_space_ship.py is in your python path or go in the example
+directory. This script should generate a cpacs file out.xml that contains a space ship
+in your current directory.
+
+Remark, this is the new way to access the python API. The old way that 
+use only tigl3wrapper.py is considered as deprecated.
+This new internal API give you access not only to the "standard" C API, but also to the functions 
+and structures of TiGL.  
```

Comment:
Noooo! tigl3wrapper is not deprecated, you need it to open the cpacs file!
You are even using it in your example.

Better:
Recently, we extended the TiGL's Python to access the entire object structure of TiGL. These classes and methods allow more control compared to the high level functions of tigl3wrapper. 
These allow for example:
  - Accessing all geometrical shapes of TiGL
  - Use, modify or visualize these shapes directly with PythonOCC
  - Use geometrical algorithms for own projects, such as Gordon's curve network interpolation method

Reply:
"Use only" was the point, I was thinking about installing manually the tigl3wrapper file. But I agree that is confusing.